tests/bin/getuid.c

```diff
@@ -0,0 +1,10 @@
+#include "libc.h"
+
+int main(void) {
+    uid_t uid = 0xFFFFFFFF;
+    uid = getuid();
+    if(is_enarx()) {
+        return uid != 1000;
+    }
+    return uid == 0xFFFFFFFF;
```

Comment:
I think this workflow is simpler. Otherwise, everything looks good.

```suggestion
    if (!is_enarx())
        return 0;

    return getuid() != 1000;
```

Reply:
Done